docs/application-connector/08-09-renew-client-cert.md

```diff
@@ -4,6 +4,8 @@ type: Tutorials
 ---
 
 By default, a client certificate you generate when you connect an external solution to Kyma is valid for 92 days. Follow this tutorial to renew a client certificate.
+ 
+> **NOTE:** You as the user are responsible for rotating and renewing the certificates.
```

Comment:
```suggestion
> **NOTE:** You, as the user, are responsible for rotating and renewing the certificates.
```

Reply:
As discussed, it's a tempting change but I understand if you decide to skip the commas.

Reply:
It should emphasize that it's the USER's responsibility (with the stress on the user), so let's go without them :)